### PR TITLE
cli53: add page

### PR DESCRIPTION
**Pull request by rod-stuchi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
cli53 version 0.8.22

### CODE REVIEW

File: pages/common/cli53.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Export a bind zone file to stdout:
+
+`cli53 export {{mydomain.com}}`
+
+- Create a `www` subdomain pointing to a relative record in the same zone:
+
+`cli53 {{rc|rrcreate}} {{mydomain.com}} {{'www 300 CNAME lb'}}`
+
+- Create a `www` subdomain pointing a external address (must ends with a dot):
```
Comment (gutjuri) [inline pages/common/cli53.md:22]:
```suggestion
- Create a `www` subdomain pointing an external address (must end with a dot):
```

File: pages/common/cli53.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`cli53 {{rc|rrcreate}} {{mydomain.com}} {{'www 300 CNAME lb.externalhost.com.'}}`
+
+- Create a `www` subdomain pointing to an IP:
```
Comment (gutjuri) [inline pages/common/cli53.md:26]:
```suggestion
- Create a `www` subdomain pointing to an IP address:
```

**Comment by rod-stuchi** [comment]
> Thank you, I've added some suggestions.

Hi, thank you for the improvement tips

File: pages/common/cli53.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Export a bind zone file to `stdout`:
+
+`cli53 export {{mydomain.com}}`
+
+- Create a `www` subdomain pointing to a relative record in the same zone:
+
+`cli53 {{rc|rrcreate}} {{mydomain.com}} {{'www 300 CNAME lb'}}`
+
+- Create a `www` subdomain pointing an external address (must end with a dot):
```
Comment (kbdharun) [inline pages/common/cli53.md:22]:
```suggestion
- Create a `www` subdomain pointing to an external address (must end with a dot):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Pull request merged**